scripts/classes/app-configurator.js

```diff
@@ -84,12 +89,19 @@ class AppConfigurator {
 
   prepareExtensions() {
     const extensions = getExtensionsFromConfiguration(this.configuration);
-    const localExtensions = getLocalExtensions(this.buildConfig.workingDirectories);
+    const linkedExtensions = getLocalExtensions(this.buildConfig.linkedExtensions);
```

Comment:
Can you use the same name for this concept. You are now using linked extensions, and local extensions in some cases.